models/all/src/index.ts

```diff
@@ -260,6 +261,16 @@ export default function buildModel (enabled: string[] = ['*'], disabled: string[
       }
     ],
     [supportModel, supportId],
+    [
+      surveyModel,
+      surveyId,
+      {
+        // TODO: icon, label, description?
```

Comment:
What are these props for? They do not affect the application entry in sidebar (which is controlled by its own `Application` model instance), so why are they?

Reply:
I think these supposed to be used to show app name and icon in settings / configuration

Reply:
Yes, this is for configuration, so entire application could be turned on/off per workspace.

Reply:
✅